aeroprom.conf.dev

```diff
@@ -28,17 +29,28 @@ db_port=3000
 
 # certificate file
 cert_file=""
+
 # key file
 key_file=""
+
+# Passphrase for encrypted key_file. Supports below formats,
+# 1. Passphrase directly                 - "<passphrase>"
+# 2. Passphrase via file                 - "file:<file-that-contains-passphrase>"
+# 3. Passphrase via environment variable - "env:<environment-variable-that-holds-passphrase>"
+key_file_passphrase="${AS_KEY_FILE_PASSPHRASE}"
```

Comment:
Not sure this should ever be set in the config file; just from either the ENV or as a command line parameter.

Reply:
Resolving as discussed. We follow the same convention in server and other tools.